src/redux/reducers/todo.reducer.js

```diff
@@ -0,0 +1,17 @@
+import * as AT from '../action.types';
+
+const INITIAL_STATE = {dataStream: {label: 'Instagram'}, rangeDate: {startDate: {}, endDate: {}}};
```

Comment:
the select component expect an array for the values so the initial data stream is for ex - 
[{ vlaue: '', label: '' }]